iree/compiler/Dialect/HAL/Target/LLVM/LLVMAOTTarget.cpp

```diff
@@ -471,17 +462,17 @@ void registerLLVMAOTTargetBackends(
   LLVMInitialize##TargetName##AsmPrinter(); \
   LLVMInitialize##TargetName##AsmParser();
 
-  static TargetBackendRegistration dylibRegistration("dylib-llvm-aot", [=]() {
+  auto backendFactory = [=]() {
     INIT_LLVM_TARGET(X86)
     INIT_LLVM_TARGET(ARM)
     INIT_LLVM_TARGET(AArch64)
     INIT_LLVM_TARGET(RISCV)
-    return std::make_unique<LLVMAOTTargetBackend>(queryOptions());
-  });
-  static TargetBackendRegistration wasmRegistration("wasm-llvm-aot", [=]() {
```

Comment:
Same here, I was using this target registration for `-iree-hal-target-backends=wasm-llvm-aot`. Fine to drop it and add back later (esp. when there are no tests verifying that it does what we expect)